x-pack/plugins/cases/public/components/all_cases/all_cases_generic.tsx

```diff
@@ -0,0 +1,408 @@
+/*
```

Comment:
This was `../cases/public/components/all_cases/index.tsx`

Look at the [diffchecker here](https://www.diffchecker.com/F8jH2472), you'll thank me later (and ill thank you back for the thorough review)

Reply:
Thank you for that!

Reply:
I knew you'd thank me later 🥳